site/src/components/pages/dashboard/dashboard-card/dashboard-card.tsx

```diff
@@ -0,0 +1,221 @@
+            }}
+          >
+            {title}
+          </Typography>
+          <Typography sx={{ mb: 2 }}>{description}</Typography>
+          <LinkWrapper title={link.title} />
+        </Box>
+      </Box>
+    </CardWrapper>
+  );
+};
+
+export const DashboardCard: FunctionComponent<
+  DashboardCardProps | LoadingCardProps
+> = (props) => {
+  const { loading, variant = "primary" } = props;
+  if (loading) {
+    return <DashboardCardLoading variant={variant} />;
+  }
+
+  const {
+    title,
+    colorGradient,
+    colorGradientOnHover,
+    description,
+    link,
+    icon,
+    CustomIcon,
+  } = props;
+
+  if (variant === "secondary") {
+    const secondaryCardProps = {
+      title,
+      description,
+      link,
+      variant,
+      icon,
+      CustomIcon,
+    };
+    return <DashboardCardSecondary {...secondaryCardProps} />;
+  }
+
+  return (
+    <CardWrapper href={link.href} onClick={link.onClick}>
+      <Box
+        sx={[
+          cardContainerSharedStyles,
+          (theme) => ({
+            // @todo update theme config to include microShadow
+            background: ({ palette }) => palette.common.white,
+            boxShadow:
+              "0px 4px 11px rgba(39, 50, 86, 0.02), 0px 2.59259px 6.44213px rgba(39, 50, 86, 0.04), 0px 0.5px 1px rgba(39, 50, 86, 0.15)",
+            borderRadius: 2,
+            flexDirection: "column",
+            border: `1px solid ${theme.palette.gray[20]}`,
+            transition: theme.transitions.create("border-color"),
+
+            "&:hover": {
+              borderColor: theme.palette.gray[30],
+
+              ".gradient:after": {
+                opacity: 1,
+              },
+            },
+
+            "&::after": {
+              boxShadow: 2,
+            },
+          }),
+        ]}
+      >
+        <Box
+          className="gradient"
+          sx={{
+            background: colorGradient,
+            borderRadius: 2,
+            borderBottomLeftRadius: 0,
+            borderBottomRightRadius: 0,
+            position: "relative",
+            transform: "scaleX(-1)",
+
+            "&::after": {
+              content: `""`,
+              position: "absolute",
+              inset: 0,
+              borderRadius: "inherit",
+              background: colorGradientOnHover,
+              opacity: 0,
+              transition: ({ transitions }) => transitions.create("opacity"),
+            },
+          }}
+          height={8}
+        />
+        <Box
+          p={4}
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            flex: 1,
+          }}
+        >
+          <Typography
+            sx={{
+              fontFamily: "Apercu Pro",
+              fontSize: "28.128px",
+              lineHeight: "120%",
+              color: "#37434F",
+              mb: 1,
+            }}
+          >
+            {title}
+          </Typography>
+          <Typography mb={2} color="#4D5C6C">
```

Comment:
we should pick colors from the theme

Reply:
Same as above

Reply:
Updated at https://github.com/blockprotocol/blockprotocol/pull/511/commits/b0e108599e1954e95939a7b15acce487c24f45fd